core/block_svg.js

```diff
@@ -523,14 +524,21 @@ Blockly.BlockSvg.prototype.tab = function(start, forward) {
 
 /**
  * Handle a mouse-down on an SVG block.
- * @param {!Event} e Mouse down event.
+ * @param {!Event} e Mouse down event or touch start event.
```

Comment:
Hmmm, yeah. Just felt a little unfortunate to me that we're throwing touch events into something specifically named for the mouse. just to throw more options in: onMouseDownOrTouchStart? onPointerStart?

but obviously nbd to me..